cmd/tegola/cmd/cache.go

```diff
@@ -357,11 +357,15 @@ func sendTiles(zooms []uint, c chan *slippy.Tile) error {
 
 		maxZoom := zooms[len(zooms)-1]
 
-		upperLeft := slippy.NewTileLatLon(maxZoom, bounds[1], bounds[0], 0, tegola.WebMercator)
-		bottomRight := slippy.NewTileLatLon(maxZoom, bounds[3], bounds[2], 0, tegola.WebMercator)
+		corner := slippy.NewTileLatLon(maxZoom, bounds[1], bounds[0], 0, tegola.WebMercator)
+		diagCorder := slippy.NewTileLatLon(maxZoom, bounds[3], bounds[2], 0, tegola.WebMercator)
 
-		_, xi, yi := upperLeft.ZXY()
-		_, xf, yf := bottomRight.ZXY()
+		_, xi, yi := corner.ZXY()
+		_, xf, yf := diagCorder.ZXY()
+
+		// ensure the initials are smaller than finals
+		if xi > xf {xi, xf = xf, xi}
+		if yi > yi {yi, yf = yf, yi}
```

Comment:
should be:

if yi > yf {yi, yf = yf, yi}

Reply:
oops, thank you for catching that

Reply:
yw...I hate when I do that sort of thing.  I was stuck for 2 hours the other day trying to figure out why the httpPlatformhandler wasn't do *anything*...turned out I had named my config file, "web..config" instead of "web.config".  The font was kinda small and I didn't notice.  DOH!